viewto-frontend/src/components/LayersPanel/LayersPanel.tsx

```diff
@@ -1,15 +1,127 @@
+import { Input } from '@strategies/react-form';
+import { useStores } from "@strategies/stores";
+import Stores from "../../stores/Stores";
+import { FiInfo } from 'react-icons/fi';
 
+const sampleData = ['data1', 'data2', 'data3', 'data4']; //todo: This needs to be replaced with the actual data
 export default observer(function LayersPanel() {
+
+    const { view } = useStores<Stores>();
     return (
         <Panel
             className="Layers"
             active={true}
         >
             <Title>Layers</Title>
             <Body>
+                <div className="Widget">
+                    <div className="_View">
+                        <div className="SelectContainer">
+                            <div className="Title">
+                                {"View Focus"} <FiInfo/>
+                            </div>
+                            <Select
+                                placeholder={'Select view focus ...'}
+                                isCreatable={true}
+                                isMulti={true}
+                                options={sampleData}
+                                value={view.viewFocus}
+                                onChange={(values: string[]) => view.setViewFocus(values)}
+                                menuPortalTarget={document.body}
+                                menuShouldScrollIntoView={false}
+                                styles={{
+                                    menuPortal: (base) => ({
+                                        ...base,
+                                    })
+                                }}
+                                menuPosition={'absolute'}
+                                menuPlacement={'auto'}
+                            />
+                        </div>
+                        <div className="SelectContainer">
+                            <div className="Title">
+                                {"View Condition"} <FiInfo/>
+                            </div>
+                            <Select
+                                placeholder={'Select view condition ...'}
+                                isCreatable={true}
+                                options={sampleData}
+                                value={view.viewCondition}
+                                onChange={(value: string) => view.setViewCondition(value)}
+                                menuPortalTarget={document.body}
+                                menuShouldScrollIntoView={false}
+                                styles={{
+                                    menuPortal: (base) => ({
+                                        ...base,
+                                    })
+                                }}
+                                menuPosition={'absolute'}
+                                menuPlacement={'auto'}
+                            />
+                        </div>
+
+                    </div>
+                    <div className="_Points">
```

Comment:
This _Points section could be cleaned up with a dictionary for the title/value pairings and a map into InputContainer